ledger/src/blocktree.rs

```diff
@@ -1847,6 +1847,34 @@ pub fn create_new_ledger(ledger_path: &Path, genesis_config: &GenesisConfig) ->
 
     blocktree.insert_shreds(shreds, None, false)?;
     blocktree.set_roots(&[0])?;
+    // Explicitly close the blocktree before we create the archived genesis file
+    drop(blocktree);
+
+    let archive_path = ledger_path.join("genesis.tar.bz2");
+    let mut args = vec!["jcfhS"];
+    args.push(archive_path.to_str().unwrap());
+    args.push("-C");
+    args.push(ledger_path.to_str().unwrap());
+    args.push("genesis.bin");
+    args.push("rocksdb");
+    let output = std::process::Command::new("tar")
```

Comment:
Yes, mostly this is duplicate code with `snapshot.tar.bz2` generation code in `snapshot_packager_service.rs`. But, I think this code duplication is tolerated because of expected infrequency of further duplication.

Reply:
Also, I omitted to add corresponding assertions to `test_create_new_ledger` because malfunction of this code can definitely be spotted by other (local_cluster, sanity) tests, while not causing hard-to-track-down test failures in unit tests due to this is rather a functionality needed at the system integration level.

Anyway, no strong preference, just time constraints. So, please let me know adding assertions are preferred.